node/tool.js

```diff
@@ -0,0 +1,44 @@
+const { Pool } = require('pg');
+const crypto = require('crypto');
+String.prototype.underline = function () {
+    return this.replace(/-/g, '_')
+}
+global.sql = (literals, ...params) => {    // For the syntax highlight in VSCode plugin: SQL Lit
```

Comment:
any reason to use `global` here?

Reply:
sql`SELECT... ` is a syntax highlight sugar, so I make it global.